segments/reminder.p9k

```diff
@@ -0,0 +1,71 @@
+# vim:ft=zsh ts=2 sw=2 sts=2 et fenc=utf-8
+################################################################
+# @title powerlevel9k Segment - Reminder
+# @source [powerlevel9k](https://github.com/bhilburn/powerlevel9k)
+##
+
+################################################################
+# Register segment
+# Parameters:
+#   segment_name  context  background  foreground  Generic  Flat/Awesome-Patched  Awesome-FontConfig  Awesome-Mapped-FontConfig  NerdFont
+#                                                                                                                             
+p9k::register_segment 'REMINDER' '' green "$DEFAULT_COLOR" '!!' ''   $'\UF0A4 '   '\u'"${CODEPOINT_OF_AWESOME_HAND_O_RIGHT}"   $'\UF0A4'
+
+################################################################
+# @description
+#   Display the reminder icon/text for the current time interval.
+##
+# @args
+#   $1 string Alignment - left | right
+#   $2 integer Segment index
+#   $3 boolean Whether the segment should be joined
+##
+prompt_reminder() {
+	typeset -AH __lookup
+  __lookup=(
+    "sun" 0
+    "mon" 1
+    "tue" 2
+    "wed" 3
+    "thu" 4
+    "fri" 5
+    "sat" 6
+  )
+	local dow=$( date +"%w" )
+	local now=$( date +"%H%M" )
+  local segment_content=""
+	local separator=""
+	for rule in $P9K_REMINDER_RULES; do
+		echo $rule | while read date_exp time_exp icon; do
```

Comment:
Interesting way of reading in variables. What would happen if the icon is empty?